### PR TITLE
device-wide quick benchmark nb + experiments: bell and Tphi

### PR DESCRIPTION
**Pull request by mitchell-brad-ibm**
Added notebook and code for batching Bell test and parallel Tphi experiments  for device wide characterization

### CODE REVIEW

**Comment by dcmckayibm** [comment]
I added a state tomography notebook that goes over all edges, https://github.com/qiskit-community/qiskit-device-benchmarking/blob/main/notebooks/bell_state_tomography.ipynb, can you look at the graph functions you added here and see if they exist already in https://github.com/qiskit-community/qiskit-device-benchmarking/blob/main/qiskit_device_benchmarking/utilities/graph_utils.py and if they don't add them?

**Comment by dcmckayibm** [comment]
Currently this is how the code runs over all edges

```
#build a set of gates
G = gu.build_sys_graph(nq, coupling_map)
#get all length 2 paths in the device
paths = rx.all_pairs_all_simple_paths(G,2,2)
#flatten those paths into a list from the rustwork x iterator
paths = gu.paths_flatten(paths)
#remove permutations
paths = gu.remove_permutations(paths)
#convert to the coupling map of the device
paths = gu.path_to_edges(paths,coupling_map)
#make into separate sets
sep_sets = gu.get_separated_sets(G, paths, min_sep=2)```

File: qiskit_device_benchmarking/bench_code/bell/partition.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
+# from qiskit-experiments-internal/qiskit_experiments_internal/tiled_experiment/partition.py
+"""
+Functions to partition nodes and edges into groups, such that nodes/edges
+in a group are distant from each other
+"""
+
+import rustworkx as rx
+
+
+def build_coupling_graph(backend, multigraph=False):
```
Comment (dcmckayibm) [inline qiskit_device_benchmarking/bench_code/bell/partition.py:10]: this is here https://github.com/qiskit-community/qiskit-device-benchmarking/blob/main/qiskit_device_benchmarking/utilities/graph_utils.py#L95

**Comment by mitchell-brad-ibm** [comment]
I changed the notebook to use the graph utils module, let me know if there's anything else needed :)

File: qiskit_device_benchmarking/bench_code/bell/partition.py (hunk 1 above)
Comment (mitchell-brad-ibm) [reply qiskit_device_benchmarking/bench_code/bell/partition.py]: removed this module and using the graph utils block like in your bell state tomo notebook

**Comment by dcmckayibm** [comment]
Can you just add this code block to the end of the notebook

```import datetime
from IPython.display import HTML, display

def qiskit_copyright(line="", cell=None):
    """IBM copyright"""
    now = datetime.datetime.now()

    html = "<div style='width: 100%; background-color:#d5d9e0;"
    html += "padding-left: 10px; padding-bottom: 10px; padding-right: 10px; padding-top: 5px'>"
    html += "<p>&copy; Copyright IBM 2017, %s.</p>" % now.year
    html += "<p>This code is licensed under the Apache License, Version 2.0. You may<br>"
    html += "obtain a copy of this license in the LICENSE.txt file in the root directory<br> "
    html += "of this source tree or at http://www.apache.org/licenses/LICENSE-2.0."

    html += "<p>Any modifications or derivative works of this code must retain this<br>"
    html += "copyright notice, and modified files need to carry a notice indicating<br>"
    html += "that they have been altered from the originals.</p>"
    html += "</div>"
    return display(HTML(html))

qiskit_copyright()
```

And a title with a small description in the beginning, e.g.

`## Bell, T1, T2 Parallel Experiment`
` This is an experiment to xxx `

**Comment by dcmckayibm** [comment]
Thanks Brad

**Pull request merged**